submissions/YuraZagor/dom tsk/index.js

```diff
@@ -0,0 +1,57 @@
+import { data } from './data.js'  
+
+const body = document.querySelector('body')
+let text = data[0].content
+
+const div = document.createElement('div')
+div.classList.add('wrapper')
+
+body.appendChild(div)
+
+const main = document.createElement('main')
+main.classList.add('main')
+div.appendChild(main)	
+
+const header = document.createElement('header')
+header.classList.add('header')
+main.appendChild(header)
+
+const content = document.createElement('div')
+content.classList.add('content')
+main.appendChild(content)
+
+
+const footer = document.createElement('footer')
+footer.classList.add('footer')
+main.appendChild(footer)
+
+let li
```

Comment:
Why do you need this variable?

Reply:
if I do not declare this variable before using event listener, the program wouldn't work and throw "  li is not defined at ul.onclick "

Reply:
This variable is not needed.

Reply:
I got rid of it, now it's like 
ul.onclick = function(event) {
	if(!event.target.closest('li')) return;
	handleClick(event.target.closest('li').id)
}
had to make a change to handleClick(li.id)